recipes/newrelic/infrastructure/logs/debian-logs.yml

```diff
@@ -98,55 +98,58 @@ install:
+
+                  # Increment found log files count
+                  foundLogFilesCount=$((foundLogFilesCount+1))
+                fi
+              done
+            fi
 
-                # Increment found log files count
-                foundLogFilesCount=$((foundLogFilesCount+1))
-              fi
-            done
-          fi
+            discoveredLogFiles=$(echo -n "{{.NR_DISCOVERED_LOG_FILES | trim}}")
+            discoveredLogsStringLength=${#discoveredLogFiles}
 
-          discoveredLogFiles=$(echo -n "{{.NR_DISCOVERED_LOG_FILES | trim}}")
-          discoveredLogsStringLength=${#discoveredLogFiles}
+            # If the user provides log files that don't exist and we haven't discovered any log files,
+            # we cannot proceed with the installation.
+            if [ $foundLogFilesCount -eq 0 ] && { [ $discoveredLogsStringLength -eq 0 ] || [ "$discoveredLogFiles" = "logs: []" ]; }; then
+              echo "ERROR Provided log files do not exist: $logFiles. No log file(s) found or discovered. Cannot proceed. Exiting installation." >> /dev/stderr
+              exit 16
+            fi
 
-          # If the user provides log files that don't exist and we haven't discovered any log files,
-          # we cannot proceed with the installation.
-          if [ $foundLogFilesCount -eq 0 ] && { [ $discoveredLogsStringLength -eq 0 ] || [ "$discoveredLogFiles" = "logs: []" ]; }; then
-            echo "ERROR Provided log files do not exist: $logFiles. No log file(s) found or discovered. Cannot proceed. Exiting installation." >> /dev/stderr
-            exit 16
-          fi
+            # Disable glob expansion (necessary for discovered logs)
+            set -f
 
-          # Disable glob expansion (necessary for discovered logs)
-          set -f
+            # index is incremented to facilitate dynamic name values for each discovered log file path
+            index=0
+            if [ $discoveredLogsStringLength -gt 0 ]; then
+              echo "logs:" | sudo tee /etc/newrelic-infra/logging.d/discovered.yml > /dev/null
 
-          # index is incremented to facilitate dynamic name values for each discovered log file path
-          index=0
-          if [ $discoveredLogsStringLength -gt 0 ]; then
-            echo "logs:" | sudo tee /etc/newrelic-infra/logging.d/discovered.yml > /dev/null
+              for filePath in $(echo -n $discoveredLogFiles | sed "s/,/ /g"); do
+                echo -e "  - name: configured-logs-$index\n    file: $filePath" | sudo tee -a /etc/newrelic-infra/logging.d/discovered.yml > /dev/null
+                index=$((index+1))
+              done
+            fi
 
-            for filePath in $(echo -n $discoveredLogFiles | sed "s/,/ /g"); do
-              echo -e "  - name: configured-logs-$index\n    file: $filePath" | sudo tee -a /etc/newrelic-infra/logging.d/discovered.yml > /dev/null
-              index=$((index+1))
-            done
+            # Enable infra-agent logs to New Relic
```

Comment:
@noahmmcgivern this whole section (Enable infra-agent logs to New Relic) is now to be removed (already removed on main).
This was causing a large ingest (too verbose logging). We now rely on the default agent configuration for those values.